go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/docker/go-units v0.4.0
 	github.com/emicklei/go-restful/v3 v3.8.0
 	github.com/fsnotify/fsnotify v1.6.0
-	github.com/gogo/protobuf v1.3.2
+	github.com/gogo/protobuf v1.3.2 // indirect
```

Comment:
nit, we usually move indirect deps in the next "indirect" collection to make it clearer

Reply:
Hmm maybe my Go toolchain is old? Let me check.